commit 771e57f3b2c19f4e6f867c01c2457ec87531b4cf
Author: Kai Uwe Broulik <kde@privat.broulik.de>
Date:   Tue Jan 31 15:07:42 2017 +0100

    [Folder View] show script execution prompt when clicking item
    
    When clicking a file in FolderView, we should ask for whether to open or run a script depending on user settings.
    This is what Dolphin does.
    
    BUG: 375793
    FIXED-IN: 5.8.6
    
    Differential Revision: https://phabricator.kde.org/D4367

diff --git a/containments/desktop/plugins/folder/foldermodel.cpp b/containments/desktop/plugins/folder/foldermodel.cpp
index 758f7f35..1ba12450 100644
--- a/containments/desktop/plugins/folder/foldermodel.cpp
+++ b/containments/desktop/plugins/folder/foldermodel.cpp
@@ -509,7 +509,8 @@ void FolderModel::run(int row)
         url.setScheme(QStringLiteral("file"));
     }
 
-    new KRun(url, 0);
+    KRun *run = new KRun(url, 0);
+    run->setShowScriptExecutionPrompt(true);
 }
 
 void FolderModel::rename(int row, const QString& name)
